README.md

```diff
@@ -36,6 +36,27 @@ In JetBrains Rider the text editor settings can be set in `Settings > Editor > F
 
 To turn off git autoCrlf, run `git config --global core.autocrlf false` in a terminal.
 
+# Secret config settings
+Certain features of the system rely on having API keys and secrets available (as do some tests).
+
+Talk to the project team to get these secrets for you to store in the secrets manager.
+
+## Secrets manager

```

Comment:
Copied this straight from https://softwiretech.atlassian.net/wiki/spaces/HEE/pages/19902234654/Secret+config+settings